addOns/encoder/src/main/java/org/zaproxy/addon/encoder/EncodeDecodeOptions.java

```diff
@@ -35,16 +35,19 @@ public class EncodeDecodeOptions extends VersionedAbstractParam {
 
     private static final String PARAM_BASE64_CHARSET = "encoder.base64charset";
     private static final String PARAM_BASE64_DO_BREAK_LINES = "encoder.base64dobreaklines";
+    private static final String PARAM_HASHERS_DO_LOWERCASE = "encoder.hashers.lowercase";
```

Comment:
I didn't roll this to options version 2 because it was adding a value and didn't need upgrade/handling.